### PR TITLE
vox guidebook update poison regen

### PR DESCRIPTION
**Pull request by Errant-4**
## About the PR
Updates vox entry in guidebook for  #33722

## Media

image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

### CODE REVIEW

**Review by VasilisThePikachu — CHANGES_REQUESTED** [review]
Should maybe specify it applies below 20 poison?

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (ArtisticRoomba) [inline Resources/ServerInfo/Guidebook/Mobs/Vox.xml:20]: Line break on sentences instead of just randomly throughout the sentence for easier review and future editing.

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (ArtisticRoomba) [inline Resources/ServerInfo/Guidebook/Mobs/Vox.xml:17]: As previously said, players should know the poison damage threshold.

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (ArtisticRoomba) [inline Resources/ServerInfo/Guidebook/Mobs/Vox.xml:12]: Same here. Avoid random line breaks in the middle of sentences. Do line breaks between them.

**Review by ArtisticRoomba — CHANGES_REQUESTED** [review]
Players should know the exact damage threshold as it's just good to know.

I'm not a maintainer, but it's good to line break between sentences and not in the middle of them. This just makes it easier to review and less weird.

IDEs and GitHub can do text wrapping for .xml files if you still want to see the text without scrolling to the left and right all the time.

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (Errant-4) [reply Resources/ServerInfo/Guidebook/Mobs/Vox.xml]: Fair

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (Errant-4) [reply Resources/ServerInfo/Guidebook/Mobs/Vox.xml]: Done

**Comment by Errant-4** [comment]
Updated the pic with regen threshold

**Review by VasilisThePikachu — APPROVED** [review]

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (ArtisticRoomba) [inline Resources/ServerInfo/Guidebook/Mobs/Vox.xml:12]:
```suggestion
  Unfortunately, space stations tend to be full of oxygen, so Vox must use an internal nitrogen supply at almost all times to avoid fatal exposure.
```

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (ArtisticRoomba) [inline Resources/ServerInfo/Guidebook/Mobs/Vox.xml:18]:
```suggestion
  Vox [color=#1e90ff]slowly recover from low levels of poison damage[/color] on their own, so long as they are careful not to exceed 20 poison damage.
```

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (ArtisticRoomba) [inline Resources/ServerInfo/Guidebook/Mobs/Vox.xml:20]:
```suggestion
  This allows them to endure breathing station air for up to thirty seconds at a time without lasting damage, letting them quickly eat, drink, take oral medication, and so on.
```

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (Errant-4) [reply Resources/ServerInfo/Guidebook/Mobs/Vox.xml:12]: I think it's fine as is. If you feel strongly about it, submit a PR with the change to my working branch, and I promise I will genuinely consider it, so long as there is time to do so before merge. But that could happen at any moment

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (Errant-4) [reply Resources/ServerInfo/Guidebook/Mobs/Vox.xml:18]: as above

File: Resources/ServerInfo/Guidebook/Mobs/Vox.xml
Comment (Errant-4) [reply Resources/ServerInfo/Guidebook/Mobs/Vox.xml:20]: as above

**Pull request merged**
